src/main/flight/rpm_filter.c

```diff
@@ -89,6 +90,7 @@ void rpmFilterInit(const rpmFilterConfig_t *config, const timeUs_t looptimeUs)
     rpmFilter.minHz = config->rpm_filter_min_hz;
     rpmFilter.maxHz = 0.48f * 1e6f / looptimeUs; // don't go quite to nyquist to avoid oscillations
     rpmFilter.fadeRangeHz = config->rpm_filter_fade_range_hz;
+    rpmFilter.fadeRangeHzInv = config->rpm_filter_fade_range_hz == 0 ? 0.0f : (1.0f / config->rpm_filter_fade_range_hz);
```

Comment:
I still think this changes the behavior because the attenuation by harmonic group is now dependent on `rpm_filter_fade_range_hz` being 0 or not. You need additional check in the loop to keep the old behavior but maybe that kills the speed improvements again...

Reply:
Oops, my bad. This should work. I need to think about it a bit longer but it should be good